api/app/insight_api.py

```
@@ -0,0 +1,152 @@
+        return response.json()
+
+    def address_transactions(self, address):
+        '''Get the transactions for an address'''
+        url = f"{self.api_url}/txs/?address={address}"
+        response = requests.get(url)
+        return response.json()
+
+    def address_utxos(self, address):
+        '''Get the unspent outputs for an address'''
+        url = f'{self.api_url}/addr/{address}/balance'
+        response = requests.get(url)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)